### DISCUSSION TITLE
Bug, can not get the dns zone list in function app website

### DISCUSSION THREAD

**Discussion by qinrui777** [polymind-inc/acmebot#265] [Q&A] [answered]
**Describe the bug**
Can not get dns zone from function app website

![image](https://user-images.githubusercontent.com/37825938/108480110-7f099d80-72d1-11eb-9d37-abae590302e2.png)

**To Reproduce**
1. Create a new function app with  "For Azure cloud "(https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fshibayan%2Fkeyvault-acmebot%2Fmaster%2Fazuredeploy.json) ,get the link from the github repo readm
2. add settings ,  set "Create new Key vault" to "true"
3. finished step ""Enable App Service Authentication"
4. go to the function app  website URL with Chrome https://func-xxxx.azurewebsites.net/add-certificate, See error

**Environment (please complete the following information):**
 - Certificate Type: None
 - Certificate Deploy Target: None

**Additional context**
When I try to revert to commit (Remove default Azure DNS configuration (#248) ), use the template json ,then create another function app, it can work properly.

**Comment by shibayan** [comment, marked as answer]
Not a bug. Similar #260 #263

This will happen if you skip Step 4.
https://github.com/shibayan/keyvault-acmebot#4-add-settings-for-your-choice-dns-provider

**Reply by qinrui777** [reply]
Actually I have done the Step4(4. Add settings for your choice DNS provider), but still error.

By the way, when I use the previous commit, I skipped the Step 4, but it can work normally.

**Reply by shibayan** [reply]
The only difference between the commits you mention is the presence or absence of `Acmebot:AzureDns:SubscriptionId`. Therefore, if you use Azure DNS as described in the README, setting `Acmebot:AzureDns:SubscriptionId` should work fine.

Due to the freedom of choice of DNS provider, we are not going to revert that commit. Please check your configuration again.
